oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/search/ClearFiltersButton.jsx

```diff
@@ -25,9 +33,7 @@ const ClearFiltersButtonComponent = ({
         onClick={() =>
           updateQueryState({
             ...currentQueryState,
-            filters: filters.filter((f) =>
-              initialFilters.map((f) => f[0]).includes(f[0])
-            ),
+            filters: filters.filter((f) => allFiltersToIgnore.includes(f[0])),
```

Comment:
Don't we want all filters in query state & just hide/ignore it from ActiveFilters?